CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,105 @@
+# Salesforce Open Source Community Code of Conduct
```

Comment:
Do we need to talk to somebody in legal before we publish it? Also, companies regularly update the code of conduct and publish it online. I think we should just refer to that link.

Reply:
These should have already been in here, they're just from the oss-template repo as instructed. It's part of the OSS process:

> Setup Your New Repository
> Squash your git commits, push your code to your repo, create the license file and add the license headers, add a code of conduct file and a security file to the project.

Reply:
got it! Is this file publicly visible and can have a legal binding? If yes, I suggest having our public code of conduct link instead of the text in this file.

Reply:
Th instructions are to add the files to project.

Reply:
Thanks!